assets/js/lib/model/users.js

```diff
@@ -1,5 +1,7 @@
 import { getFromConfig } from '@lib/config';
 
 const TRENTO_ADMIN_USERNAME = getFromConfig('adminUsername') || 'admin';
+const OIDC_ENABLED = getFromConfig('oidcEnabled') || false;
 
 export const isAdmin = (user) => user.username === TRENTO_ADMIN_USERNAME;
+export const isSingleSignOnEnabled = () => OIDC_ENABLED;
```

Comment:
TBH I don't know if lib/model/users is the right place to put SSO stuff, it seems also strange in tests checking and mocking for an app configuration in the users model

Maybe a file in @lib/auth called config or something like that is more appropriate?

Wdyt?

Reply:
Yes, I prefer it in fact.
Maybe `sso.js` of `singlesignon.js`.
I will think about it.
I will move things around

Reply:
Done